store/copr/coprocessor.go

```diff
@@ -612,7 +612,7 @@ func chooseBackoffer(ctx context.Context, backoffermap map[uint64]*Backoffer, ta
 	if ok {
 		return bo
 	}
-	newbo := backoff.NewBackofferWithVars(ctx, copNextMaxBackoff, worker.vars)
+	newbo := backoff.NewBackofferWithVars(ctx, CopNextMaxBackoff, worker.vars)
```

Comment:
why not use failpoint ?

Reply:
I don't think it's particularly necessary to create a new "failpoint". This is just changing the visibility
Why do you think "failpoint" is better?

Reply:
To avoid the variant be modified unexpectedly in the future.